AndroidXCI/lib/src/main/kotlin/dev/androidx/ci/testRunner/ApkStore.kt

```diff
@@ -117,5 +145,33 @@ internal class ApkStore(
         return null
     }
 
+    suspend fun getUploadedAdditionalApk(
```

Comment:
another alternative, seems like maybe all we want is an overload to `uploadApk` that will receive `GcsPath` instead of `ByteArray`? otherwise, is there anythign else special about additional APKs?